test/ErrorRecovery/ABI/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -0,0 +1,2 @@
+
+ERROR;
```

Comment:
:-) Could you maybe add a comment that this is meant to simulate a parsing error of cling's original RuntimePrintValue.h?